### PR TITLE
[DOCS] Clarify multi-tenancy behavior of rules being isolated per space

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1023.

Preview [here](https://security-docs_1072.docs-preview.app.elstc.co/guide/en/security/master/security-spaces.html).

### CODE REVIEW

File: docs/getting-started/security-spaces.asciidoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+under the `.siem-signals-<spacename>-*` index pattern, and they may be
+accessed by any user with role privileges to access those
+{es} indices. In our example above, any user with {es} privileges to access
+`.siem-signals-SOC_prod-*` will be able to view `SOC_prod` alerts from
+within {es} and other {kib} apps such as Discover and Lens. 
+
+To ensure that detection alert data remains private to the space in which
+it was created, ensure that the roles to which your security solution users
+are assigned include {es} privileges that limit their access to only alerts
+within their space alerts index.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/getting-started/security-spaces.asciidoc:34]: Yes, I think that would help.
```suggestion
within their space's alerts index. 
```

File: docs/getting-started/security-spaces.asciidoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+under the `.siem-signals-<Kibana-space>-*` index pattern, and they may be
+accessed by any user with role privileges to access those
+{es} indices. In our example above, any user with {es} privileges to access
+`.siem-signals-SOC_prod-*` will be able to view `SOC_prod` alerts from
+within {es} and other {kib} apps such as Discover and Lens. 
+
+To ensure that detection alert data remains private to the space in which
+it was created, ensure that the roles assigned to your {elastic-sec} users
+include {es} privileges that limit their access to only alerts
+within their space's alerts index.
```
Comment (jmikell821) [inline docs/getting-started/security-spaces.asciidoc:34]:
```suggestion
To ensure that detection alert data remains private to the space in which
it was created, ensure that the roles assigned to your {elastic-sec} users
include {es} privileges that limit their access to alerts
within their space's alerts index. 
```

**Review by jmikell821 — APPROVED** [review]
Tiny nit. LGTM otherwise.

**Comment by jmikell821** [comment]
I would actually probably place this right after System Reqs - but I'll let others weigh in.

File: docs/getting-started/security-spaces.asciidoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+`SOC_prod` space.
+
+[NOTE]
+===== 
+By default, alerts created by detection rules are stored in {es} indices
+under the `.siem-signals-<Kibana-space>-*` index pattern, and they may be
+accessed by any user with role privileges to access those
+{es} indices. In our example above, any user with {es} privileges to access
+`.siem-signals-SOC_prod-*` will be able to view `SOC_prod` alerts from
+within {es} and other {kib} apps such as Discover and Lens.
```
Comment (spong) [inline docs/getting-started/security-spaces.asciidoc:29]: ++ Thanks for the clarification here! Was my only remaining question after reading the above.

**Review by spong — APPROVED** [review]
LGTM! Thanks @joepeeples! 🙂

**Review by MikePaquette — APPROVED** [review]
LGTM, thanks!

**Pull request merged**